docs/Telemetry.md

```diff
@@ -138,6 +138,9 @@ Refer to the below table to understand different broadcast events raised during
 | `InitiateEndChatOnBrowserUnload`      | End active chats on browser unload |
 | `ClosePopoutWindow`      | Event to close popout window  |
 | `RaiseErrorEvent`                 | On raising error events |
+| `NetworkDisconnected`                 | On network Disconnects |
+| `NetworkReconnected`                 | On network reconnects |
+| `HideChatVisibilityChangeEvent`                 | On hiding the chat window or tab |
```

Comment:
This is not something chat-widget send, but receives. We don't need to log it here